scripts/vagrant_ubuntu_provision.sh

```
@@ -0,0 +1,31 @@
+    libwavpack-dev \
+    ninja-build \
+    pkg-config \
+    libopusfile-dev \
+    libxmp-dev \
+    fluidsynth
+
+# Install steamcmd
+sudo add-apt-repository multiverse; sudo dpkg --add-architecture i386; sudo apt update
+sudo apt install steamcmd
+
+[[ -d breakhack ]] || git clone --recursive https://github.com/oliveshark/breakhack
```

Shellcheck (0.11.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)